reverse_strand.py

```diff
@@ -26,14 +28,14 @@
     name = read.query_name
     info = name.split("|")
     nucl = info[3]
+    if read.is_unmapped: # Can be decoded with bitwise flag with & 4 (4 means unmapped)
```

Comment:
Minor style comment: per PEP 8 style guide, comment should be separated by two spaces from the code, e. g. :

```python
    if read.is_unmapped:  # Can be decoded with bitwise flag with & 4 (4 means unmapped)
```

(Here and throughout the code)